src/wooden-tree/stories/WoodenTree.stories.js

```diff
@@ -8,6 +8,7 @@ import { generator, flatLazyChildren } from './Generator';
 import { ReduxTree } from './components/ReduxTree';
 import { ConnectedNode } from './components/ReduxNode';
 import combinedReducers from './reducers';
+import '../index.scss';
```

Comment:
I'm not sure about this line though, is this just a demo?

Reply:
Yass